test/extended/sqlsrv_AE_functions.inc

```diff
@@ -320,9 +320,14 @@ function connect($attestation_info)
         // See https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/configure-always-encrypted-enclaves?view=sqlallproducts-allversions#configure-a-secure-enclave
         $query = "SELECT [name], [value], [value_in_use] FROM sys.configurations WHERE [name] = 'column encryption enclave type';";
         $stmt = sqlsrv_query($conn, $query);
-        $info = sqlsrv_fetch_array($stmt);
-        if ($info['value'] != 1 or $info['value_in_use'] != 1) {
-            die("Error: enclave computations are not enabled on the server!");
+        if (!$stmt) {
+            print_r(sqlsrv_errors());
+            die("Error when checking if enclave computations are enabled. This should never happen!\n");
+        } else {
+            $info = sqlsrv_fetch_array($stmt);
+            if (empty($info) or ($info['value'] != 1) or ($info['value_in_use'] != 1)) {
+                die("Error: enclave computations are not enabled on the server!");
+            }
```

Comment:
I'd rather check it here, so that the test clearly fails if enclave computations are not enabled (the skipif checks that the server is a HGS enabled VM - if it is, then these tests should proceed and work as expected).